docs/_docs/templates.md

```diff
@@ -530,13 +638,33 @@ site. If you use `linenos`, you might want to include an additional CSS class
 definition for the `.lineno` class in `syntax.css` to distinguish the line
 numbers from the highlighted code.
 
-### Link
+### Gist
+
+Use the `gist` tag to easily embed a GitHub Gist onto your site. This works
```

Comment:
This is not a Jekyll tag. It is [provided by a plugin](https://github.com/jekyll/jekyll-gist), and [not installed by default](https://github.com/jekyll/jekyll/blob/v3.3.1/jekyll.gemspec#L32-L41). It's documentation probably doesn't belong here.